tests/test_traceback_log.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2019 The Jaeger Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import mock
+import sys
+from jaeger_client import Span, SpanContext
+
+
+def test_traceback_full():
+    """Test that a traceback is logged with both location and message"""
+    mock_tracer = mock.MagicMock()
+    mock_tracer.max_tag_value_length = 300
+    context = SpanContext(trace_id=1, span_id=2, parent_id=None, flags=1)
+    span = Span(context=context, operation_name='traceback_test', tracer=mock_tracer)
+
+    try:
+        raise ValueError('Something unexpected happened!')
```

Comment:
what about using context manager properties of the span to catch the exception automatically?
```
with span:
    raise X
```
The base class implements it: https://github.com/opentracing/opentracing-python/blob/master/opentracing/span.py#L218

Reply:
That's a good point, I've made that change to tests.